### DISCUSSION TITLE
MITM - Fiddler "Auto responder"

### DISCUSSION THREAD

**Discussion by accessT0ken** [chimurai/http-proxy-middleware#743] [Q&A]
### Describe the feature you'd love to see

It's possible to do
// http://www.example.org/api/foo/bar ->http://localhost:3000/api/foo/bar
not
// http://localhost:3000/api/foo/bar -> http://www.example.org/api/foo/bar
? Like replace example.org page with my page from localhost?

**Comment by chimurai** [comment]
Not sure what you're try to do. Can you clarify?

**Reply by accessT0ken** [reply]
I'm trying to make something like a Fiddler "Auto responder"

**Reply by chimurai** [reply]
Think that's beyond the scope of this project.
